geoids/geoid_intro.md

```diff
@@ -0,0 +1,60 @@
+
+Imagine suspending a plumb line (a string with a weight) in a flat area. The string points directly toward the center of gravity at that location. Now place the same plumb line next to a massive mountain. The mountain's mass exerts a gravitational pull of its own, slightly attracting the weight toward it. This means the plumb line no longer points directly toward the Earth's center but is subtly deflected. This deflection alters what we perceive as "vertical," which in turn affects how we define "level" surfaces like the geoid.
+
+So, in areas near large mountains, the geoid bulges slightly due to the extra mass, and in areas over ocean trenches or low-density rock, it dips. The result is a globally continuous but irregular surface, one that responds to the Earth’s gravity field rather than its geometric shape alone.
+
+## Why is the geoid important?
+
+Because GPS provides heights relative to a smooth ellipsoid, ignoring gravity, the geoid is critical for converting those heights into meaningful elevations that relate to sea level. Whether you're designing a levee, mapping terrain, or predicting flood extents, understanding the true vertical reference (i.e. the geoid) ensures accurate and consistent height measurements worldwide.
+
+## How do we measure it?
+
+The geoid cannot be measured directly but is inferred by combining:
+
+* Satellite gravimetry missions (e.g., [GRACE](https://grace.jpl.nasa.gov/mission/grace/), [GOCE](https://www.esa.int/Enabling_Support/Operations/GOCE)) to capture large-scale gravity variations.
+* Terrestrial and airborne gravity surveys (e.g., NOAA gravity data) for finer regional detail. https://www.youtube.com/watch?v=2phuKuj-SII
+* Precise GNSS measurements (ellipsoidal heights) at known benchmarks.
+* Traditional leveling data linking physical benchmarks and tide-gauge observations.
+
+These datasets feed into gravity field models and boundary-value problems to compute geoid undulations (height differences between a reference ellipsoid and the geoid surface). The result is a gridded geoid height model giving, at each location, the amount to subtract from GNSS-measured ellipsoid heights to get orthometric heights.
+
+## Ellipsoids and relationship to geoids
+
+* Reference ellipsoids (e.g., GRS80, WGS84) provide smooth mathematical surfaces for defining latitude/longitude and ellipsoidal heights.
+* The geoid height at a location is the difference between the ellipsoid and the geoid. Positive values mean the geoid lies above the ellipsoid, negative means below.
+* Converting a GNSS ellipsoidal height (h) to a more meaningful elevation (H) uses: H = h − N, where N is the geoid height from a model aligned to the same ellipsoid.
```

Comment:
Let's use equation formatting here. That way we can reference it elsewhere, and also it'll look better as PDF https://mystmd.org/guide/math#equations 

I think a figure would be great here. Or at least state the range (e.g. the PSU reference above has a nice sentence "In fact, the separation between the lumpy surface of the geoid and the smooth GRS80 ellipsoid worldwide varies from about +85 meters west of Ireland to about -106 meters, the latter in the area south of India near Ceylon.")